debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -151,7 +151,7 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
             }
 
             SlotCreationResult slotCreatedInfo = null;
-            if (snapshotter.shouldStream()) {
+            if (snapshotter.shouldStream() || YugabyteDBServer.isEnabled()) {
```

Comment:
Will it make more sense if we add another block before this instead of modifying this which just creates a replication slot if we have the `initial_only` snapshotter enabled? Probably something like:
```java
if (YugabyteDBServer.isEnabled() && !snapshot.shouldStream()) {
  // Create replication slot.
}
```

Reply:
Have added an extra check to create a slot in YB case only if an existing slot is not present.